ddtrace/contrib/internal/openai/_endpoint_hooks.py

```diff
@@ -262,7 +262,7 @@ def _record_request(self, pin, integration, instance, span, args, kwargs):
     def _record_response(self, pin, integration, span, args, kwargs, resp, error):
         resp = super()._record_response(pin, integration, span, args, kwargs, resp, error)
         if kwargs.get("stream") and error is None:
-            return self._handle_streamed_response(integration, span, kwargs, resp, is_completion=False)
+            return self._handle_streamed_response(integration, span, kwargs, resp, operation_type="chat")
         integration.llmobs_set_tags(span, args=[], kwargs=kwargs, response=resp, operation="chat")
```

Comment:
Some upstream changes mean we're going to need to handle some conflicts. @lievan should be able to help you out with this